gcp/tests/test_connect_scheduling.py

```
@@ -0,0 +1,562 @@
+class TestParseDateConnect:
+    """parse_date coerces BQ string/date/datetime to date."""
+
+    def test_string_iso(self):
+        assert parse_date("2026-06-10") == date(2026, 6, 10)
+
+    def test_date_passthrough(self):
+        d = date(2026, 6, 10)
+        assert parse_date(d) is d
+
+    def test_datetime_extracts_date(self):
+        dt = datetime(2026, 6, 10, 9, 0, tzinfo=timezone.utc)
```

Ruff (0.15.15)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)